### DISCUSSION TITLE
Drag and Drop on collision

### DISCUSSION THREAD

**Discussion by antoinechamot** [formkit/formkit#1375] [General]
Hi,
I want to activate drag and drop on collision. When dragged object touch another one. Is it possible ? i don't see any example on the website.
Thanks

**Comment by fenilli** [comment]
It is not possible but it is on the works something similar that would work for this.

**Comment by sashamilenkovic** [comment]
@antoinechamot We have a separate https://github.com/formkit/drag-and-drop/discussions here. 

I'm not sure what you two mean by "activate drag and drop on collision". Do you have an example you can share so I can learn? Do you mean including some sort of styling when you drag an item over another?

**Reply by fenilli** [reply]
I think he means exactly what `https://github.com/formkit/drag-and-drop/discussions/65`, programatically enable/disable the drop function when drag enter/drag exit happens on other available containers.

**Comment by antoinechamot** [comment]
What i mean : 
When i drag and drop cards. I want to react not when the mouse pointer enters the drop zone but when the edge on my dragged card collide with another card. For example something like : https://konvajs.org/docs/sandbox/Collision_Detection.html

**Reply by sashamilenkovic** [reply]
@antoinechamot Ah, gotcha. There is no way currently to do this for @formkit/drag-and-drop. You can play around with adjusting the `threshold` property of the config (the percentage either horizontally or vertically you must move your mouse into a drop zone before the sort operation is performed). Also, we have a separate FormKit Drag and Drop discussion here: https://github.com/formkit/drag-and-drop/discussions. Having collision detection is a good idea though, would you mind filing an issue/starting a discussion about it on the drag and drop repo?

**Reply by antoinechamot** [reply]
I have open an issue : https://github.com/formkit/drag-and-drop/issues/68
